scripts/start/submit.R

```diff
@@ -48,17 +48,28 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
       # detected like this
       firstRunInCascade <- normalizePath(renv::project()) == normalizePath(".")
       if (firstRunInCascade) {
-        capture.output({ # suppress printing status
-          renvStatus <- renv::status()
-        })
-        if (!renvStatus$synchronized) {
-          message("The new run will use the package environment defined in renv.lock, ",
-                  "but it is out of sync, probably because you installed packages/updates manually. ",
-                  "Write current package environment into renv.lock first? (Y/n)", appendLF = FALSE)
-          if (tolower(gms::getLine()) %in% c("y", "")) {
-            renv::snapshot(prompt = FALSE)
+          capture.output({ # suppress printing status
+              renvStatus <- renv::status()
+          })
+
+          if (!renvStatus$synchronized) {
+              # check environment variable on whether to syncronise renv
+              Rrs <- Sys.getenv('REMIND_sync_renv')
```

Comment:
Also for consistency, I'd suggest we use an option for this, just like for the autoRenvUpdates:
```suggestion
              Rrs <- getOption("remindSyncRenv", "ask")
```